src/strategies/zkportal/index.ts

```diff
@@ -0,0 +1,39 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import fetch from 'cross-fetch';
+
+export const author = 'vicsn';
+export const version = '0.1.0';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+
+  const url = `https://snapshot.zkportal.io/verify_eth_addresses`;
```

Comment:
Hey @vicsn you own this API? if so, I think it is better to use https://github.com/snapshot-labs/snapshot-strategies/tree/master/src/strategies/api-post strategy in your space directly. to avoid duplication. but is there some reason for a separate strategy or you don't have access to change this API? let me know :)

Reply:
Hi Chaitu, thx for the quick response. Makes sense, I didn't know the other strategy existed! We will actually build in signature verification into this strategy as well, to ensure DAO users require less trust in our backend infra. Will close this PR and open it when signature verification is added.